controlpanel/api/models/user.py

```diff
@@ -44,11 +44,11 @@ def get_id_token(self):
 
     @property
     def iam_role_name(self):
-        return f"{settings.ENV}_user_{self.username.lower()}"
+        return cluster.User(self).iam_role_name
```

Comment:
Long term (as in, not as part of this), and assuming `iam_role_name` is mainly used in the `cluster` module we may well get rid of these from the models.

(maybe these properties are used in some of the models now to grant/revoke access by passing the target ARN but we may well pass the whole bucket to `(User|App).(grant|revoke)_access()` if necessary)